tools/qontract_cli.py

```diff
@@ -349,6 +349,40 @@ def integrations(ctx):
     print_output(ctx.obj['output'], environments, columns)
 
 
+@get.command()
+@click.pass_context
+def mirrors(ctx):
```

Comment:
there are other mirrors... should we be specific here? I'd expect the command to be `get quay-mirrors`

Reply:
+1